app/src/components/tables/BlockTable/BlocksTable.tsx

```diff
@@ -250,36 +250,39 @@ export const BlocksTable: React.FC = () => {
       sorting={[
         {
           id: blocksTableOptions.sorting.sortBy,
-          desc: blocksTableOptions.sorting.order === 'desc',
+          desc: blocksTableOptions.sorting.order === TableOrder.Descending,
```

Comment:
See above comment. This seems to me to be overkill unless you have good reason it use an enum here?